app/templates/index.html

```diff
@@ -90,17 +89,13 @@ <h1>'Allo, 'Allo!</h1>
       r.parentNode.insertBefore(e,r)}(window,document,'script','ga'));
       ga('create','UA-XXXXX-X');ga('send','pageview');
     </script>
-
-    <!-- build:js scripts/vendor.js -->
-    <!-- bower:js -->
-    <!-- endbower -->
-    <!-- endbuild -->
+    <!-- build:js scripts/vendor.js -->	
+    <!-- endbuild -->	
+    <% if (includeJQuery || includeBootstrap) { %>
+      <script type="text/javascript" src="/node_modules/jquery/dist/jquery.min.js"></script>
```

Comment:
I think jQuery should be within `build.js` comments. Even though this one is already compressed, useref _concatenates_ all vendor scripts into one `vendor.js` file, resulting in a single HTTP request, which is what we're aiming for.

Reply:
Yes! Sounds fine to me